vxquery-core/src/main/java/org/apache/vxquery/compiler/rewriter/rules/ConvertDocExpressionToFile.java

```diff
@@ -0,0 +1,171 @@
+/*
```

Comment:
I can't help feel this file needs a better name. Maybe replaceSourceMapInDocExpression?